apps/studio/components/interfaces/Organization/InvoicesSettings/InvoicesSettings.tsx

```diff
@@ -85,72 +95,86 @@ export const InvoicesSettings = () => {
     )
   }
 
+  // Handle loading state faded text for table headers
+  const tableHeadClassName =
+    isLoading || invoices.length === 0 ? 'text-foreground-muted' : undefined
+
   return (
-    <div className="overflow-hidden md:overflow-auto overflow-x-scroll">
-      <Table
-        head={[
-          <Table.th key="header-icon" />,
-          <Table.th key="header-date">Date</Table.th>,
-          <Table.th key="header-amount">Amount</Table.th>,
-          <Table.th key="header-invoice">Invoice number</Table.th>,
-          <Table.th key="header-status" className="flex items-center">
-            Status
-          </Table.th>,
-          <Table.th key="header-download" className="text-right"></Table.th>,
-        ]}
-        body={
-          isLoading ? (
+    <Card>
+      <Table>
+        <TableHeader>
+          <TableRow>
+            {invoices.length > 0 && (
+              <TableHead className="w-2">
+                <span className="sr-only">Icon</span>
```

Comment:
Makes the word “Icon” read by screen readers. Otherwise it’s just an ambitious `thead`.